app/views/collections/_meta.html.erb

```diff
@@ -1,17 +1,19 @@
 <% title title_string %>
-<link rel="canonical" href="<%= app_url("/about") %>" />
-<meta name="description" content="<%= title_string %> on <%= community_name %>">
+
+<link rel="canonical" href="<%= app_url(path) %>" />
+<meta name="description" content="<%= description_string %>">
 <%= meta_keywords_default %>
 
 <meta property="og:type" content="article" />
-<meta property="og:url" content="<%= app_url("/about") %>" />
-<meta property="og:title" content="About <%= community_name %>" />
+<meta property="og:url" content="<%= app_url(path) %>" />
+<meta property="og:title" content="<%= title_string %>" />
 <meta property="og:image" content="<%= SiteConfig.main_social_image %>" />
-<meta property="og:description" content="<%= community_name %> is great!" />
+<meta property="og:description" content="<%= description_string %>" />
 <meta property="og:site_name" content="<%= community_name %>" />
 
 <meta name="twitter:card" content="summary_large_image">
 <meta name="twitter:site" content="@<%= SiteConfig.social_media_handles["twitter"] %>">
-<meta name="twitter:title" content="About <%= community_name %>">
-<meta name="twitter:description" content="<%= community_name %> is great!">
+<meta name="twitter:title" content="<%= title_string %>">
+<meta name="twitter:creator" content="@<%= @user.twitter_username %>">
```

Comment:
This should be optional, `.twitter_username` can be empty as a user could not have a Twitter username

Reply:
Fixed, thanks for the help!